Changes

```diff
@@ -1,5 +1,6 @@
 {{$NEXT}}
 
+- #15: Fixed issue where save wouldn't work if called during global destruction
```

Comment:
Mark that as "Issue 15", otherwise folk are going to be wondering what the "15" means. :)